website/docs/plugins/ax_sweeper.md

```diff
@@ -62,6 +62,8 @@ ax.modelbridge.dispatch_utils: Using Bayesian Optimization generation strategy:
 
 In this example, we set the range of `x` parameter as an integer in the interval `[-5, 5]` and the range of `y` parameter as a float in the interval `[-5, 10.1]`. Note that in the case of `x`, we used `int(interval(...))` and hence only integers are sampled. In the case of `y`, we used `interval(...)` which refers to a floating-point interval. Other supported formats are fixed parameters (e.g.` banana.x=5.0`), choice parameters (eg `banana.x=choice(1,2,3)`) and range (eg `banana.x=range(1, 10)`). Note that `interval`, `choice` etc. are functions provided by Hydra, and you can read more about them [here](https://hydra.cc/docs/next/advanced/override_grammar/extended/). An important thing to remember is, use [`interval`](https://hydra.cc/docs/next/advanced/override_grammar/extended/#interval-sweep) when we want Ax to sample values from an interval. [`RangeParameter`](https://ax.dev/api/ax.html#ax.RangeParameter) in Ax is equivalent to `interval` in Hydra. Remember to use `int(interval(...))` if you want to sample only integer points from the interval. [`range`](https://hydra.cc/docs/next/advanced/override_grammar/extended/#range-sweep) can be used as an alternate way of specifying choice parameters. For example `python example/banana.py -m banana.x=choice(1, 2, 3, 4)` is equivalent to `python example/banana.py -m banana.x=range(1, 5)`.
 
+Tagging the interval with `log` will tell Ax to sample in log space (e.g. `python example/banana.py -m banana.x=tag(log, interval(1, 1000))`). When specifying a range sweep in a config file, include `log_scale: true` to achieve the same.
+
```

Comment:
I think it would be helpful to give a concrete example for how to use `log_scale` in a config file. Would it make sense to move this paragraph later in the docs so that it comes after the [banana yaml example](https://github.com/facebookresearch/hydra/blob/e7ad04b300aba0c59d65cf1384e2f6f8a677cd8e/website/docs/plugins/ax_sweeper.md?plain=1#L68-L76)? That way, we could include the following to demonstrate usage of `log_scale`:
```yaml
banana.y:
 type: range
 bounds: [-5, 10.1]
 log_scale: true
```

Reply:
Looks good!